go-controller/pkg/node/gateway_init_linux_test.go

```diff
@@ -34,6 +34,8 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+var primaryLinkName = "primary"
```

Comment:
should we take the opportunity to change it to a constant? or it needs to be a var?

Reply:
It does not. I can change it and relaunch the CI to check if it passes now.

Reply:
it's a nit, just realized while reviewing this, maybe is a good first issue to report?, `switch vars to constants whenever is possible`?

Reply:
Sure, I don't see why not.